src/jupyterhub_cost_monitoring/const_cost_aws.py

```diff
@@ -6,7 +6,7 @@
 
 # Environment variables based config isn't great, see fixme comment in
 # values.yaml under the software configuration heading
-CLUSTER_NAME = os.environ["CLUSTER_NAME"]
+CLUSTER_NAME = os.environ.get("CLUSTER_NAME", "")
```

Comment:
with `os.environ.get("CLUSTER_NAME", "")` things may fail silently down the line if `CLUSTER_NAME` is not defined. I would propose we keep `os.environ["CLUSTER_NAME"]` which fails explicitly when `CLUSTER_NAME` isn't defined and set a dummy value for tests if required.

Reply:
fabc2aa